netbox_branching/database.py

```diff
@@ -21,11 +21,6 @@ def _get_db(self, model, **hints):
             warnings.warn(f"Routing database query for {model} before branching support is initialized.")
             return
 
-        # Bail if the model does not support branching
```

Comment:
I'm assuming this isn't needed anymore, won't affect plugins and such? Didn't know how to test that.

Reply:
Correct; this was originally addressed under #59 but isn't technically needed.